docsite/rst/intro_configuration.rst

```diff
@@ -144,6 +144,27 @@ different locations::
 
 Most users will not need to use this feature.  See :doc:`developing_plugins` for more details
 
+.. _stdout_callback:
+
+stdout_callback
+===============
+
+.. versionadded:: 2.0
+
+This setting allows you to override the default stdout callback for ansible-playbook.
+
+.. _callback_whitelist:
+
+callback_whitelist
+==================
+
+.. versionadded:: 2.0
+
+Now ansible ships with all included callback plugins ready to use but they are disabled by default,
+this setting lets you enable a list of additional callbacks, this cannot change or override the
+default stdout callback, use :ref:`stdout_callback` for that.
+
+
 .. _command_warnings:
```

Comment:
Maybe an example for both of these.  Is stdout_callback just a True|False to enable or disable it?  Is callback_whitelist just a comma separated string of plugin names?  Do the plugin names end in ".py"?